ncm-metaconfig/src/main/metaconfig/chrony/pan/schema.pan

```diff
@@ -0,0 +1,85 @@
+declaration template metaconfig/chrony/schema;
+
+include 'pan/types';
+
+type chrony_service_server_flag = choice(
+    'iburst', 'burst', 'offline', 'auto_offline', 'prefer', 'noselect', 'trust', 'require', 'xleave');
+
+@documentation{
+    the crony.conf configuration entry for server/pool/peer options
+}
+type chrony_service_server_options = {
+    'minpoll' ? long = -6 # power of 2 (1/64th of a second)
+    'maxpoll' ? long(0..) = 6 # power of 2 (32)
+    'key' ? string
+    'maxdelay' ? double
+    'maxdelayratio' ? double
+    'maxdelaydevratio' ? double
+    'mindelay' ? double
+    'asymmetry' ? double(-1..1)
+    'offset' ? double
+    'minsamples' ? long(0..)
+    'maxsamples' ? long(0..)
+    'maxsources' ? long = 4
+    'filter' ? long(0..)
+    'polltarget' ? long(6..60) = 8
+    'port' ? type_port
+    'presend' ? long(0..)
+    'minstratum' ? long
+    'version' ? choice('3', '4') = 4
```

Comment:
NB, default value has not effect for an optional field unless it is explicitly set to undef.

Reply:
Ah I didn't know default value for optional fields have ANY effect, so I used it merely as documentation/comment, will change this